pod-operation/src/components/lim_temperature.rs

```diff
@@ -21,9 +34,10 @@ impl LimTemperature {
 		self.ads1015.destroy_ads1015();
 	}
 
-	pub fn read_pins(&mut self) -> (i16, i16, i16, i16) {
+	pub fn read_pins(&mut self) -> (f32, f32, f32, f32) {
 		[SingleA0, SingleA1, SingleA2, SingleA3]
 			.map(|channel| block!(self.ads1015.read(channel)).unwrap())
+			.map(voltage_to_temp)
```

Comment:
Praise: thanks for maintaining the functional approach, should make it easier to swap out the measurement portion if needed (e.g. when exploring async options)
Note: chaining maps (for arrays rather than iterators) is not officially recommended, but the array has only four values, so this should be okay ([`array::map`](https://doc.rust-lang.org/std/primitive.array.html#method.map))